File: Java programming/problems on string manipulation/problem116.java
/*
116. Write a program which accept string from user and copy that characters of that 
		string into another string in reverse order. 
	
		Input :   "Marvellous Python" 
		Output :   "nohtyP suollevraM" 
	
		Author : Milind Nalawade
	
		Date : 16/07/2021
	
		Language : Java 14.0.2
*/

import java.lang.*;
import java.util.*;

class Logic
{
	public static void Output(String arg,StringBuilder arr)
	{
		for(int i=(arg.length())-1;i>=0;i--)
		{
			arr.append(arg.charAt(i));
		}
	}	
}
class problem116 extends Logic
{
	public static void main(String args[])
	{
		Scanner sobj = new Scanner(System.in);
		System.out.print("\n Enter your string : ");
		String str=sobj.nextLine();
		StringBuilder arr=new StringBuilder(str.length());
		Output(str,arr);
		System.out.println("\n Output : "+arr);
	}
}
	

File: C++ Programming/problems on string manipulation/problem116.cpp
/*

116. Write a program which accept string from user and copy that characters of that string into another string in reverse order. 
	
	Input :   “Marvellous Python” 
	Output :   “nohtyP suollevraM” 
	
	Author : Milind Nalawade
	
	Date : 16/07/2021
	
	Language : C++ programming
	
*/

using namespace std;
#include<iostream>

string Stringreverse(string& str)
{
    int no=str.length();
    int j=0;
    char brr[30];
    for(int i=no-1;i>=0;i--)
    {
        brr[j]=str[i];
        j++;
    }
    return string(brr);
}

int main()
{
	string str;
	string arr;
	cout<<"Enter your string : ";
	cin>>str;
	arr=Stringreverse(str);
	cout<<"\n Output : "<<arr;
	return 0;
}